tests/conftest.py

```diff
@@ -15,8 +15,11 @@
 import pytest
```

Comment:
What is tests/asdf.json??

Reply:
I thought I had removed when I pushed the commit, guess it came back accidentally. It was a json file just to make some json more readable.